src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,62 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "moviesessions")

```

Comment:
Lets separate movie and session words.

Reply:
@flekros please, pay attention to this comment